tests/metrics/collect_mem_consmd.sh

```diff
@@ -128,38 +140,100 @@ function start_collect()
 # It kills the collectl process.
 function stop_collect()
 {
+	echo "Stop collectl"
 	kill "$COLLECT_PID"
 }
 
-# Get the average about a "csv" file produced
+# It launches a instance of smem tool
+# in order to get metrics about proportional
+# set size.
+function start_pss_monitor()
+{
+	bash ./smem_monitor.sh $PROC &
+	SMEM_PID=$!
+}
+
+# It terms smem process
+function stop_pss_monitor()
+{
+	kill "$SMEM_PID"
+}
+
+# Get PSS memory metrics
+function get_pss_metrics()
+{
+	pss_sum=0
+	pss_count=0
+	samples=""
+
+	while read line; do
+		pss_mem=$(echo "$line" | sed 's/[A-Za-z]*//g')
+		res=$(echo "$pss_mem > 0" | bc )
```

Comment:
It would be a good idea to check that `bc` is available before starting this script. Something like:

```
cmd=bc
command -v "$cmd" 2>/dev/null || { echo "ERROR: command $cmd not available"; exit 1; }

```

You could make that clearer if you defined a `die` function:

```
die()
{
    local msg="$*"
    echo >&2 "ERROR: $msg"
    exit 1
}

    :
    :

cmd=bc
command -v "$cmd" 2>/dev/null || die "command $cmd not available"
```